examples/horovod/horovod.json

```diff
@@ -0,0 +1,32 @@
+{
+  "jobName": "horovod-mpi-cifar10",
+  "image": "openpai/example.horovod.mpi",
+  "dataDir": "$PAI_DEFAULT_FS_URI/examples/tensorflow/distributed-cifar-10/data",
+  "outputDir": "$PAI_DEFAULT_FS_URI/examples/horovod/output",
+  "codeDir": "$PAI_DEFAULT_FS_URI/examples/horovod/code",
+  "virtualCluster": "default",
+  "retryCount": 0,
+  "taskRoles": [
+    {
+      "name": "main",
+      "taskNumber": 1,
+      "cpuNumber": 4,
+      "memoryMB": 16384,
+      "shmMB": 64,
```

Comment:
is shmMB: 64 necessary here? I remember the default value is 64?

Reply:
Yes, it's the default value. I haven't change it.

Reply:
please remove this filed then

---
In reply to: [230639934](https://github.com/Microsoft/pai/pull/1656#discussion_r230639934) [](ancestors = 230639934)

Reply:
fix